setup.py

```diff
@@ -22,9 +22,15 @@
     author="UW-IT AXDD",
     author_email="aca-it@uw.edu",
     include_package_data=True,
-    install_requires=['UW-RestClients-Core>0.8,<1.0',
+    install_requires=['UW-RestClients-Core == 0.8.0',
                       'python-dateutil',
+                      'Pillow',
                       'mock',
+                      'commonconf',
+                      'requests',
+                      'configparser',
```

Comment:
If this is only needed for travis, add a pip install configparser to the travis.yml instead.